client/src/Components/ReviewsContainer/List/ReviewList.jsx

```diff
@@ -0,0 +1,87 @@
+import React, { useState, useEffect } from "react"
+import Container from "react-bootstrap/Container"
+import Row from "react-bootstrap/Row"
+import Col from "react-bootstrap/Col"
+import Button from "react-bootstrap/Button"
+import ReviewTile from "./ReviewTile.jsx"
+import useReviews from "../../../contexts/ReviewsContext.jsx"
+
+const ReviewList = (props) => {
+  const { reviews, fetchReviews } = useReviews()
+  const [page, setPage] = useState(1)
+  const [count, setCount] = useState(2)
+  const [sort, setSort] = useState("relevant") // based on sort
+  const [product_id, setProduct_id] = useState(42367)
+  const [tiles, setTiles] = useState([])
+  const [next, setNext] = useState([])
+
+  // create first set of tiles
+  useEffect(() => {
+    fetchReviews(page, count, sort, product_id, (data) => {
+      setTiles(
+        data.results.map((review) => (
+          <Row key={review.review_id}>
+            <Col>
+              <ReviewTile {...review} />
+            </Col>
+          </Row>
+        ))
+      )
+      setPage(page + 1)
+    })
+  }, [count, sort, product_id])
+
+  // create next sets of tiles
+  useEffect(() => {
+    fetchReviews(page, count, sort, product_id, (data) => {
```

Comment:
re-writing some of the same code here for each callback, maybe pass in `setNext` or `setTiles` to be called from one fetchReviews function?